.bazelignore

```diff
@@ -15,15 +15,19 @@ app/ide-desktop/lib/ts-plugin-namespace-auto-import/node_modules
 app/ide-desktop/node_modules
 app/lang-markdown/node_modules
 app/lezer-markdown/node_modules
+app/table-expression/node_modules
 app/rust-ffi/node_modules
 app/ydoc-server/node_modules
 app/ydoc-server-nodejs/node_modules
 app/ydoc-server-polyglot/node_modules
 app/ydoc-shared/node_modules
-app/ydoc-shared/parser-codegen/generated
```

Comment:
Changed this as it didn't seem to refer to the actual output path

Reply:
It is a codegen path that we used to have, I kept it around on purpose to not break local setups right after upgrade. It is probably safe to remove now.

Reply:
Ok, but the path `app/ydoc-shared/src/ast/generated` should be listed here, right?

Reply:
It could be, though I think the generated sources are explicitly ignored in the appropriate rule anyway.

Reply:
According to a comment in the build file, the .bazelignore is the mechanism for ignoring these sources.

Reply:
I think the comment was written when it was the intention to depend on ignore file, but we ended up adding an explicit ignore to the `glob` anyway, see `IGNORED_GENERATED_FILES` usage. In that case we can probably remove the ignore file entries. Could you also update that comment to reflect the actual mechanism used?